db/migrate/20170309111313_add_resolved_to_alerts.rb

```diff
@@ -0,0 +1,5 @@
+class AddResolvedToAlerts < ActiveRecord::Migration[5.0]
+  def change
+    add_column :miq_alert_statuses, :resolved, :boolean
```

Comment:
We do not need that since resolved alerts are only used in the datawarehouse provider which is prototype and hidden in the product. Also resolved alerts are not yet collected (that pr is being reviewed as well https://github.com/ManageIQ/manageiq/pull/14299)